src/utils/tableUtil.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ColumnName, detectorDataset, detectorTable } from "../checkLinuxWebAppDownDetector";
+
+// the dataset contains several tables all with the same tableName and columnNames so to find the proper table, look for a specific value
+export function findTableByRowValue(datasets: detectorDataset[], searchValue: string): detectorTable | undefined {
```

Comment:
Feels like it would be better to have all the detector logic in one folder, rather than having a file like this in `utils`. As it is now, I would have no idea this file is for detectors based on the naming

`validateWebsite` could probably go the same folder, too.